third_party/move/move-model/src/ast.rs

```diff
@@ -569,6 +569,16 @@ pub enum RewriteResult {
     Unchanged(Exp),
 }
 
+/// Visitor position
+pub enum VisitorPosition {
+    Pre,
+    MidMutate,  // after RHS and before LHS of Mutate expression.
+    BeforeBody, // Before body of Block expression.
+    BeforeThen, // Before then clause of IfElse expression.
+    BeforeElse, // Before else clause of IfElse expression.
+    Post,
```

Comment:
```suggestion
    Post,  // after visiting all children
```

Reply:
same here, subexpressions instead of children.

Reply:
+1 for subexpressions instead of children